CONTRIBUTING.md

```diff
@@ -0,0 +1,51 @@
+# Contributing to MarkLogic-kubernetes
+
+Thank you for your interest in contributing to this project! We welcome contributions from the community to make this project better.
+
+- [Found an Issue](#found-an-issue)
+- [Want a Feature](#want-a-feature)
+- [Getting Started](#getting-started)
+- [Issue Tracker](#issue-tracker)
+- [PR management](#pr-management)
+- [Code Style](#code-style)
+
+## Found an Issue?
+
+If you find a bug in the source code or a mistake in the documentation, you can help us by submitting an issue 
+to our [GitHub Issue Tracker][Issue Tracker]. If you'd like to submit a feature enhancement, please first create an 
+issue with your proposed idea so that we can start a discussion about the problem you want to solve and what the best
+solution would be.  
+
+## Want a Feature?
+
+You can request a new feature by submitting an issue to our [GitHub Issue Tracker][Issue Tracker].  If you
+would like to implement a new feature then first create a new issue and discuss it with one of our
+project maintainers.
+
+## Getting Started
+
+To get started with contributing, please follow these steps:
+
+1. Fork the repository and clone it to your local machine.
+2. Install the necessary dependencies.
+3. Create a new branch for your changes.
+4. Make your desired changes to the codebase.
+5. Test your changes thoroughly.
+6. Tests can be done using the test framework. See [test folder](./test/) and [Makefile](makefile)
+
+## Issue Tracker
+
+If you encounter any issues or have suggestions for improvements, please open an issue in the issue tracker. Be sure to provide as much detail as possible, including steps to reproduce the issue if applicable.
+
+## PR management
+
+Created PR will not be merge as is.
+The MarkLogic docker team will use the PRs for "inspiration" but not merge the changes in directly. They may rewrite the code as they like, incorporating the submitted changes into their own code.
```

Comment:
Here you mentioned Marklogic docker team, should it be Kubernetes team or just MarkLogic team?

Reply:
Done.